### ISSUE TITLE
intel nuc image is broken?

### ISSUE THREAD

**Issue by ArgoHA** [home-assistant/operating-system#1357]
**Hardware Environment**
Intel nuc7cjyh, 4 gb ram

**Home Assistant OS release:**
Fresh installation from [this page](https://www.home-assistant.io/installation/nuc)

I downloaded fresh intel nuc image, flashed with balenaEtcher brand new ssd and installed it in brand new intel nuc7cjyh with 4 gb of ram. I could not find Home Assistant IP address in router's panel. I saw the device on ethernet connection and it had a mac address, but not an IP address. I tried another cable and even another router with another ISP - got the same thing. Installed fresh image, connected monitor - now I see "Intel Nuc" on monitor and that's it. I've been waiting for 40 minutes.
Then I tried to install ubuntu on that machine to check if hardware is fine and I succeeded. Then I tried to download archive Home Assistant image (for intel nuc) but got the exact same behaviour.

So, to recap, I am doing fresh install of Home Assistant image fir intel nuc and I do not get Home Assistant start running. All I see on monitor is "Intel Nuc"

![aFlFJ7C033I](https://user-images.githubusercontent.com/43640256/117616475-d94d2880-b173-11eb-8bac-9c1f7212771e.jpeg)

Then, if I'll try to reboot the system I would get this

![0791c7f9475e440904ec4ee3ef9fceb33aae9dca](https://user-images.githubusercontent.com/43640256/117616737-2f21d080-b174-11eb-87de-d320212a0cf0.jpeg)

**Comment by ArgoHA** [comment]
Here is the [solution](https://www.intel.com/content/www/us/en/support/articles/000038401/intel-nuc/intel-nuc-kits.html)
Just needed to turn off secure boot in bios. Seems like home assistant tutorial on installation should have that info?
Anyways, this issue can be closed
